examples/batch/argo-workflows-batch/README.ipynb

```diff
@@ -0,0 +1,400 @@
+   "source": [
+    "with open(\"assets/input-data.txt\", \"w\") as f:\n",
+    "    for i in range(10000):\n",
+    "        f.write('[[1, 2, 3, 4]]\\n')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Check the contents of the file"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "10000 assets/input-data.txt\n",
+      "[[1, 2, 3, 4]]\n",
+      "[[1, 2, 3, 4]]\n",
+      "[[1, 2, 3, 4]]\n",
+      "[[1, 2, 3, 4]]\n",
+      "[[1, 2, 3, 4]]\n",
+      "[[1, 2, 3, 4]]\n",
+      "[[1, 2, 3, 4]]\n",
+      "[[1, 2, 3, 4]]\n",
+      "[[1, 2, 3, 4]]\n",
+      "[[1, 2, 3, 4]]\n"
+     ]
+    }
+   ],
+   "source": [
+    "!wc -l assets/input-data.txt\n",
+    "!head assets/input-data.txt"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Upload the file to our minio"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 5,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "\u001b[m\u001b[32;1mBucket created successfully `minio-seldon/data`.\u001b[0m\n",
+      "...-data.txt:  146.48 KiB / 146.48 KiB ┃▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓┃ 2.03 MiB/s 0s\u001b[0m\u001b[0m\u001b[m\u001b[32;1m"
+     ]
+    }
+   ],
+   "source": [
+    "!mc mb minio-seldon/data\n",
+    "!mc cp assets/input-data.txt minio-seldon/data/"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Create Argo Workflow\n",
+    "\n",
+    "In order to create our argo workflow we have made it simple so you can leverage the power of the helm charts.\n",
+    "\n",
+    "Before we dive into the contents of the full helm chart, let's first give it a try with some of the settings.\n",
+    "\n",
+    "We will run a batch job that will set up a Seldon Deployment with 10 replicas and 100 batch client workers to send requests."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 29,
+   "metadata": {
+    "scrolled": false
+   },
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Name:                seldon-batch-process\r\n",
+      "Namespace:           default\r\n",
+      "ServiceAccount:      default\r\n",
+      "Status:              Pending\r\n",
+      "Created:             Mon Jun 08 11:57:56 +0100 (1 second ago)\r\n"
+     ]
+    }
+   ],
+   "source": [
+    "!helm template seldon-batch-workflow helm-charts/seldon-batch-workflow/ \\\n",
+    "    --set workflow.name=seldon-batch-process \\\n",
+    "    --set seldonDeployment.name=sklearn \\\n",
+    "    --set seldonDeployment.replicas=10 \\\n",
+    "    --set batchWorker.workers=100 \\\n",
+    "    --set batchWorker.payloadType=ndarray \\\n",
+    "    --set batchWorker.dataType=data \\\n",
+    "    | argo submit -"
+   ]
```

Comment:
This step fails for me
```
state:                                                                                                                                                                                                                       │
│      terminated:                                                                                                                                                                                                                │
│        containerID: containerd://14fe7109ba18cb0cacc4876c22156db91ddff2d49e80671a8d308aef3ab28851                                                                                                                               │
│        exitCode: 1                                                                                                                                                                                                              │
│        finishedAt: "2020-06-08T14:06:49Z"                                                                                                                                                                                       │
│        message: 'Error from server (Forbidden): error when creating "/tmp/manifest.yaml":                                                                                                                                       │
│          seldondeployments.machinelearning.seldon.io is forbidden: User "system:serviceaccount:default:default"                                                                                                                 │
│          cannot create resource "seldondeployments" in API group "machinelearning.seldon.io"                                                                                                                                    │
│          in the namespace "default"'                                                                                                                                                                                            │
│        reason: Error                                                                                                                                                                                                            │
│        startedAt: "2020-06-08T14:06:48Z"                                                     
```

Reply:
Kind cluster - argo installed via https://github.com/argoproj/argo/blob/master/docs/getting-started.md

Reply:
Had same issues - got slightly further executing
```
kubectl create rolebinding default-admin --clusterrole=cluster-admin --serviceaccount=default:default
```
Though not sure if `argo` likes to run in `argo` - hit problems even trying to run their hello world in `kind`.

Reply:
Oh yeah Rafal mentioned he ran into that issue, it seemed like a permissions/rbac issue. I used docker desktop and didn't come across these issues - @cliveseldon did the rolebinding resolve the issue? I can add that as a note in the Notebook in case more people have the same issue